### PR TITLE
Add person bulkCreate API

### PR DESCRIPTION
**Pull request by anjosh-zz**
example request:
```http
POST /bulkCreate
[
   {
      "birthdate":"1991-05-09",
      "email":"jstuttles@gmail.com",
      "fullname":"Jason Suttles",
      "phoneNumber":"+19192331210",
      "preferredContactMethod":"phone"
   }
]

### CODE REVIEW

**Comment by Djeisen642** [comment]
I added a couple things to this that don't affect what this does

**Comment by Djeisen642** [comment]
Can be merged

File: routes/persons.js [hunk 1]
```diff
@@ -46,6 +68,31 @@ router.post('/create', middleware.continueIfLoggedIn, async (req, res) => {
...
 
 
+router.post('/bulkCreate', async (req, res) => {
+  try {
+    let createdPersons = await models.Person.bulkCreate(req.body, {
+      individualHooks: true,
+      fields: Object.values(PERSON_FIELDS)
+    });
+
+    let persons = req.body.map((person, i) => {
```
Comment (Djeisen642) [inline routes/persons.js:78]: This is the only weird part about this. Perhaps we should match by email instead of trying to match by place in the array

**Review by Djeisen642 — CHANGES_REQUESTED** [review]

File: routes/persons.js (hunk 1 above)
Comment (anjosh) [reply routes/persons.js]: Not convinced that order is kept but looks like this example is depending on order as well.

**Comment by anjosh-zz** [comment]
Going to merge for now created issue for https://github.com/Djeisen642/htm-project-backend/pull/7#discussion_r183216641

**Pull request merged**